sdk/python/kfp/components/_data_passing.py

```diff
@@ -169,24 +169,25 @@ def get_canonical_type_for_type_struct(type_struct) -> str:
         return None
 
 
-def get_deserializer_code_for_type(typ) -> str:
-    try:
-        return type_name_to_deserializer.get(
-            get_canonical_type_struct_for_type[typ], None)
-    except:
-        return None
+# Doesn't seem to be used anymore, can be removed?
+# def get_deserializer_code_for_type(typ) -> str:
+#    try:
+#        return type_name_to_deserializer.get(
+#            get_canonical_type_struct_for_type[typ], None)
+#    except:
+#        return None
 
 
-def get_deserializer_code_for_type_struct(type_struct) -> str:
+def get_deserializer_code_for_type_struct(type_struct: str) -> str:
     try:
-        return type_name_to_deserializer.get(type_struct, None)
+        return type_name_to_deserializer.get(type_annotation_utils.get_short_type_name(type_struct), None)
```

Comment:
nit: this line exceeds the [80 characters limit](https://google.github.io/styleguide/pyguide.html#32-line-length).
Can you please format the code using YAPF: https://github.com/zillow/pipelines/blob/master/CONTRIBUTING.md#sdk